addon/components/polaris-resource-list/filter-control/date-selector.js

```diff
@@ -183,35 +182,27 @@ export default Component.extend({
         label: 'in the next year',
       },
     ];
-  }).readOnly(),
+  }
 
-  dateOptionTypes: computed(function() {
+  get dateOptionTypes() {
     return {
-      past: [
-        ...this.get('datePastOptions'),
-        ...this.get('dateComparatorOptions'),
-      ],
-      future: [
-        ...this.get('dateFutureOptions'),
-        ...this.get('dateComparatorOptions'),
-      ],
+      past: [...this.datePastOptions, ...this.dateComparatorOptions],
+      future: [...this.dateFutureOptions, ...this.dateComparatorOptions],
       full: [
-        ...this.get('datePastOptions'),
-        ...this.get('dateFutureOptions'),
-        ...this.get('dateComparatorOptions'),
+        ...this.datePastOptions,
+        ...this.dateFutureOptions,
+        ...this.dateComparatorOptions,
       ],
     };
-  }).readOnly(),
+  }
 
-  now: computed(function() {
+  get now() {
```

Comment:
Does this value get memoized the same way the CP used to? 🤔

Reply:
not without a `@computed` decorator

Reply:
Hmm OK... so we probably want dat `@computed` decorator here then to maintain behaviour